src/components/filopplaster/filopplasterModal/FilopplasterModal.tsx

```diff
@@ -121,12 +163,25 @@ const FilopplasterModal: React.FC = () => {
     try {
       const kommaTilPunktum = belopString.replace(',', '.');
       const inputBelop = parseFloat(kommaTilPunktum);
```

Comment:
[utils/skjemavalidering.ts](https://github.com/navikt/flex-reisetilskudd-frontend/blob/52c4c35597d5220ae0a6e32c0d9338b908474c86/src/utils/skjemavalidering.ts#L6) har en funksjon for å validere beløp av norske kroner (med begrensning på maks to desimaler, da én øre er laveste lovlige valør i Norge).

I Dagens transportmiddel gjøres det ingen parsing av beløpet over til float, men kan jo være at vi burde det der. Man kan jo også anta at slik parsing gjøres et annet sted, men vi bør gjøre det likt begge stedene vi håndterer nummerinputs